pkg/controllers/kube-controller-manager.go

```diff
@@ -96,6 +96,8 @@ func configure(cfg *config.MicroshiftConfig) (args []string, applyFn func() erro
 	}
 
 	args, err = mergeAndConvertToArgs(overrides)
+	args = append(args, fmt.Sprintf("-v=%d", cfg.LogVLevel))
```

Comment:
Does including
```
	"v": {cfg.LogVLevel},
```
To `ExtendedArguments` in `overrides` do the same? You wouldnt need this line.

Reply:
@pacevedom  Yup, looks better to use the same approach for both short and long arguments. I have modified `GetKubeControllerManagerArgs` to support that.